src/components/Main/Main.tsx

```diff
@@ -130,7 +130,7 @@ function Main() {
     }
   }, [autorunSimulation, debouncedRun, scenarioState, locationSearch, severity])
 
-  const [setScenarioToCustom] = useDebouncedCallback((newParams: AllParams) => {
+  const setScenarioToCustom = (newParams: AllParams) => {
```

Comment:
Remove the useDebounceCallback here because the debounce is now handled by `DebouncedDeterministicLinePlot`.